From 13e8017fe422fbc56a1269cb28503febf604ea57 Mon Sep 17 00:00:00 2001
From: Luwei Zhou <b45643@freescale.com>
Date: Tue, 10 Dec 2013 15:19:47 +0800
Subject: [PATCH 432/449] ENGR00290679 mlb: Revert "ENGR00287256 linux-test:
 Fix the SYNC mode hang issue when connected with
 MITB."

This reverts commit 946255450d230ecf3d05ee575d7f1b69fd1c5713. The old
patch provides a workaround to MLB driver hang in error interrupt
handler. The root cause has been found in the MLB driver. So revert
this patch.
---
 test/mxc_mlb_test/mxc_mlb150_test.c | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/test/mxc_mlb_test/mxc_mlb150_test.c b/test/mxc_mlb_test/mxc_mlb150_test.c
index 968f49c..a24b936 100644
--- a/test/mxc_mlb_test/mxc_mlb150_test.c
+++ b/test/mxc_mlb_test/mxc_mlb150_test.c
@@ -291,15 +291,6 @@ int do_txrx_test(int fd)
 		return -1;
 	}
 
-	if (0 == t_case) {
-		ret = ioctl(fd, MLB_IRQ_DISABLE);
-		if (ret) {
-			printf("Failed to disable the IRQ\n");
-			return -1;
-		}
-	}
-
-
 	if (blocked) {
 		while (1) {
 			ret = read(fd, buf, 2048);
@@ -386,15 +377,6 @@ int do_txrx_test(int fd)
 
 shutdown:
 	usleep(1000);
-
-	if (0 == t_case) {
-		ret = ioctl(fd, MLB_IRQ_ENABLE);
-		if (ret) {
-			printf("Failed to enalbe the IRQ\n");
-			return -1;
-		}
-	}
-
 	ret = ioctl(fd, MLB_CHAN_SHUTDOWN);
 	if (ret) {
 		printf("Failed to shutdown async channel\n");
-- 
1.8.0

